lib/ui/compositing.dart

```diff
@@ -249,6 +249,23 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// an optimization. It has no effect on the correctness of rendering.
   /// {@endtemplate}
   ///
+  /// {@template dart.ui.sceneBuilder.oldLayerVsRetained}
+  /// Passing a layer to [addRetained] or as `oldLayer` argument to a push
+  /// method counts as _usage_. A layer be used no more than once in a scene.
+  /// For example, it may not be passed simultaneously to two push methods, or
+  /// to a push method and to `addRetained`.
+  ///
+  /// When a layer is passed to [addRetained] all descendant layers are also
+  /// considered as used in this scene. The same single-usage restriction
+  /// applies to descendants.
+  ///
+  /// When a layer is passed as an `oldLayer` argument to a push method, it may
+  /// no longer be used in subsequent frames. If you would like to continue
+  /// reusing the resources associated layer, store the layer object returned
```

Comment:
Nit: "associated layer" -> "associated with the layer"?

Reply:
Done.